### DISCUSSION TITLE
too many redirects - new site install

### DISCUSSION THREAD

**Discussion by megaskills** [vanjarosoftware/Vanjaro.Platform#1945] [General]
I just installed the latest version downloaded 10 minutes ago. 
The installation worked without any issue, but when the site is launched it displays 'too many redirects'
i have tried resetting the cookie cache to no avail.

any other ideas of what to look for?

this was a brand new fresh install. the db was created, and all tables appear to be created. 
the files are located in the web folder as expected - do i need to set special folder permissions?

**Comment by meetmandeep** [comment]
How was it installed? Using the Vanjaro Installer or manually?
